tests/test_handlers.py

```diff
@@ -243,6 +243,42 @@ def test_am1_bcc():
 
     assert vjp_fn(charges_adjoints) == None
 
+def test_am1_ccc():
+    
+    patterns = [
+        ['[#6X4:1]-[#6X1,#6X2:2]', -0.0269],
+        ['[#16:1]-[#8:2]', 99.]
+    ]
+
+    smirks = [x[0] for x in patterns]
+    params = np.array([x[1]*np.sqrt(138.935456) for x in patterns])
+    props = None
+
+    am1h = nonbonded.AM1CCCHandler(smirks, params, props)
+    mol = Chem.AddHs(Chem.MolFromSmiles("CC#C"))
+    AllChem.EmbedMolecule(mol)
+    es_params, es_vjp_fn = am1h.parameterize(mol)
+
+    ligand_params = np.array([
+        -1.65,
+        -1.65,
+        -2.61,
+        1.10,
+        1.10,
```

Comment:
AM1 is still symmetric?

Reply:
they are not exactly the same, only up to 2 decimal places